src/sentry/utils/snuba.py

```diff
@@ -595,11 +595,11 @@ def snuba_query(params):
         query_params, forward, reverse, thread_hub = params
         try:
             with timer("snuba_query"):
-                body = json.dumps(query_params)
                 referrer = headers.get("referer", "<unknown>")
                 if SNUBA_INFO:
+                    logger.info("{}.body: {}".format(referrer, query_params))
```

Comment:
This no longer dumps valid JSON so we won't be able to easily copy + paste the snuba query any more. Is this fine?

Reply:
🤦 dangit you're right, included json.dumps in this format here, Wanted to keep the copy+paste ability but didn't want to introduce a second `if SNUBA_INFO` check